package.json

```diff
@@ -24,145 +24,147 @@
-    "@react-navigation/bottom-tabs": "^6.3.1",
-    "@react-navigation/drawer": "^6.6.15",
-    "@react-navigation/native": "^6.1.7",
-    "@react-navigation/native-stack": "^6.9.13",
-    "@rnmapbox/maps": "^10.1.27",
-    "@sentry/react-native": "~5.20.0",
-    "@tanstack/react-query": "^5.12.2",
-    "@turf/helpers": "^7.0.0",
-    "@turf/random": "^7.0.0",
-    "@types/luxon": "^3.4.2",
-    "assert": "^2.0.0",
-    "buffer": "^6.0.3",
-    "cheap-ruler": "^3.0.2",
-    "eventemitter3": "^5.0.1",
-    "expo": "~50.0.20",
-    "expo-application": "~5.8.4",
-    "expo-av": "~13.10.6",
-    "expo-build-properties": "~0.11.1",
-    "expo-camera": "~14.1.3",
-    "expo-crypto": "~12.8.1",
-    "expo-dev-client": "~3.3.12",
-    "expo-device": "~5.9.4",
-    "expo-document-picker": "~11.10.1",
-    "expo-file-system": "~16.0.9",
-    "expo-font": "~11.10.3",
-    "expo-localization": "~14.8.4",
-    "expo-location": "~16.5.5",
-    "expo-secure-store": "~12.8.1",
-    "expo-sensors": "~12.9.1",
-    "expo-splash-screen": "~0.26.5",
-    "expo-status-bar": "~1.11.1",
-    "expo-system-ui": "^2.9.4",
-    "expo-task-manager": "~11.7.3",
-    "expo-updates": "~0.24.13",
-    "geojson": "^0.5.0",
-    "geojson-geometries-lookup": "^0.5.0",
-    "lodash.isequal": "^4.5.0",
-    "luxon": "^3.4.4",
-    "map-obj": "^5.0.2",
-    "nanoid": "^5.0.1",
-    "nodejs-mobile-react-native": "^18.17.8",
+    "@osm_borders/maritime_10000m": "1.1.0",
+    "@react-native-community/hooks": "3.0.0",
+    "@react-native-community/netinfo": "11.4.1",
+    "@react-native-picker/picker": "2.10.2",
+    "@react-navigation/bottom-tabs": "6.5.8",
+    "@react-navigation/drawer": "6.6.15",
+    "@react-navigation/native": "6.1.7",
+    "@react-navigation/native-stack": "6.9.13",
+    "@rnmapbox/maps": "10.1.33",
+    "@sentry/react-native": "5.20.0",
+    "@tanstack/react-query": "5.62.8",
+    "@turf/helpers": "7.1.0",
+    "@turf/random": "7.1.0",
+    "@types/luxon": "3.4.2",
+    "assert": "2.1.0",
+    "buffer": "6.0.3",
+    "cheap-ruler": "4.0.0",
+    "eventemitter3": "5.0.1",
+    "expo": "50.0.21",
+    "expo-application": "5.8.4",
+    "expo-av": "13.10.6",
+    "expo-build-properties": "0.11.1",
+    "expo-camera": "14.1.3",
+    "expo-crypto": "12.8.1",
+    "expo-dev-client": "3.3.12",
+    "expo-device": "5.9.4",
+    "expo-document-picker": "11.10.1",
+    "expo-file-system": "16.0.9",
+    "expo-font": "11.10.3",
+    "expo-localization": "14.8.4",
+    "expo-location": "16.5.5",
+    "expo-secure-store": "12.8.1",
+    "expo-sensors": "12.9.1",
+    "expo-splash-screen": "0.26.5",
+    "expo-status-bar": "1.11.1",
+    "expo-system-ui": "2.9.4",
+    "expo-task-manager": "11.7.3",
+    "expo-updates": "0.24.13",
+    "geojson": "0.5.0",
+    "geojson-geometries-lookup": "0.5.0",
+    "lodash.isequal": "4.5.0",
+    "luxon": "3.5.0",
+    "map-obj": "5.0.2",
+    "nanoid": "5.0.9",
+    "nodejs-mobile-react-native": "18.20.4",
```

Comment:
for future reference, wondering if upgrading anything related to nodejs-mobile (or any runtime we use) should be left this as a separate task just because of its importance and how it could affect other parts of our setup (e.g. native prebuilds)

Reply:
Im not fully sure? For a major version, I would say yes. But for a minor/patch version, if everything is running properly, i don't know if there is a need. But I also don't have as much background on how these dependencies are interacting

Reply:
yeah i think in this case it worked out fine. would say that moving forward, we should definitely be more careful about minor+major version upgrades of nodejs-mobile